test/Microsoft.AspNet.HttpOverrides.Tests/project.json

```diff
@@ -0,0 +1,26 @@
+﻿{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "Microsoft.AspNet.HttpOverrides": "1.0.0-*",
+    "Microsoft.AspNet.Http.Abstractions": "1.0.0-*",
```

Comment:
You don't need to list any transient dependencies (e.g. anything already referenced by HttpOverrides).